internal/whois/whois.go

```diff
@@ -55,13 +56,13 @@ var (
 type whoisClient struct{}
 
 // NewClient return a "live" whois client.
-func NewClient() client.Client {
+func NewClient() whoisClient {
```

Comment:
here too... `whoisClient` is not exported... there is not much use in returning an unexported type I think...

Reply:
Replied above